src/App.test.tsx

```diff
@@ -3,7 +3,7 @@ import { render, screen } from '@testing-library/react';
 import App from './App';
 
 test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  // render(<App />);
```

Comment:
um teste comentado, n vale só apagar se n for pra ter?

Reply:
@igorcruzf então, se a gente apagar esse teste a suite passa a falhar e aí a solução seria apagar a suite (o arquivo), mas como a gente vai precisar de testes ali futuramente, optei por deixar ali comentado só para a suite não falhar

Reply:
okay!